blacktie/tx/src/main/cpp/HttpTxManager.cxx

```diff
@@ -379,6 +379,7 @@ bool HttpTxManager::handle_request(
 					LOG4CXX_DEBUG(httptxlogger, "Return 200 OK with body: " <<
 						status << " XA status: " << res);
 					code = 200;
+					_branches.erase(branch->get_name());
```

Comment:
Similarly here too - this change is wrong (see Amos' comment on the associated JIRA JBTM-2363)